driver-core/src/main/com/mongodb/operation/AsyncQueryBatchCursor.java

```diff
@@ -274,16 +299,19 @@ private BsonDocument asKillCursorsCommandDocument(final ServerCursor localCursor
                 .append("cursors", new BsonArray(singletonList(new BsonInt64(localCursor.getId()))));
     }
 
+    private void endOperationInProgress() {
+        boolean pendingClose = false;
```

Comment:
Nit: don't change the word order in a variable name of a local just to hide the fact that the name you want hides a field with the same name.

Instead, call it something like `localPendingClose`, or even just `pendingClose`, and use `this.pendingClose` to refer to the field.

Reply:
I use `pendingClose` and `this.pendingClose`.